.github/workflows/Test.yml

```
@@ -0,0 +1,71 @@
+name: Test Allrun
+on: 
+  push:
+  pull_request:
+jobs:
+  validation:
+    runs-on: Ubuntu-latest    
+    container: 
+      image: ubuntu:20.04
+    steps:
+     - name: Checkout
+       uses: actions/checkout@v3
+     - name: initialize
+       env:
+        DEBIAN_FRONTEND: noninteractive
+       run: |
+        apt install -y --no-install-recommends 
+        apt-get update 
+        apt-get install -y sudo wget unzip git software-properties-common make cmake g++ mpich openmpi-bin libopenmpi-dev libscalapack-mpi-dev 
```

Please pay attention to the usage of `--no-install-recommends`.
```suggestion
        apt-get update 
        apt-get install -y --no-install-recommends sudo wget unzip git software-properties-common make cmake g++ mpich openmpi-bin libopenmpi-dev libscalapack-mpi-dev 
```